### ISSUE TITLE
Option to lock now playing widget controls to specific app (Spotify)

### ISSUE THREAD

**Issue by toby1999** [pock/pock#409]
**Describe the feature you'd like to improve in Pock**

First of all I love Pock, I've been using it for a week now.

It would be cool if there was an option to lock the now playing controls to my preferred music player. At the moment, the now playing widget opens Apple music when I press it to start playing music. It would be great if this instead opened Spotify, and also if the now playing controls never switched to controlling safari (youtube videos) so I can always just use them for Spotify.

Tl;dr: I was thinking it would be nice to have the option for the now playing widget to just be exclusively for Spotify.

Hope this helps :)
<img width="1085" alt="Touch Bar Shot 2020-05-18 at 19 08 01" src="https://user-images.githubusercontent.com/44904736/82245885-ae9f0a00-993b-11ea-8d7e-b8bef09c4584.png">
